### PR TITLE
Implementing 4 statistics #30

### PR DESCRIPTION
**Pull request by lRolwes**
Goal: Implement 4 stats: Average Height, Number of Newtonian Systems, Average Resultant, and Number of PCF Systems
Files changed: ExploreSystems.js, Postrgress_Connector.py, Server.py, Routes.js
- Implemented Stats: Num PCF, Avg Height, Avg Resultant, Number of Newtonian, 
- Added funtion getStatisticas to Postgress Connector to calculate the statistics
- Added Function fetchStatistics ti ExploreSystems.js to use getStatistics
- Added getStatistics to routes and server.py to allow for the funtion to be used between frontend and backend over port 5000
- Added useState to ExploreSystems.js to store the statistics.
- Removed text field from stats box (not sure why it was there to begin with

Future Action: There are still more statistics to implement

### CODE REVIEW

File: Backend/server.py [hunk 1]
```diff
@@ -36,6 +36,14 @@ def data4():
     data = connector.getFilteredSystems(filters)
     return jsonify(data)
 
+# expects json describing filters, returns the systems that satisfy filters
```
Comment (AbhilashKotha) [inline Backend/server.py:39]: @lRolwes I think yo uneed to Change the comments here

**Review by Evan-Richmond — APPROVED** [review]
Address Abhilash's comment, then looks good

File: Backend/server.py (hunk 1 above)
Comment (lRolwes) [reply Backend/server.py]: Fixed, sorry about that

**Review by AJelliss — APPROVED** [review]
Looks great to me, testing well on my machine, just fix those conflicts

**Pull request merged**